web/app/view/ReportController.js

```diff
@@ -208,15 +210,45 @@ Ext.define('Traccar.view.ReportController', {
                 params: {
                     id: positionIds
                 },
+                scope: this,
                 callback: function (records, operation, success) {
                     if (success) {
                         Ext.getStore('ReportRoute').add(records);
+                        if (this.singleEvent && records.length > 0) {
```

Comment:
I think we can just select first item for any report. Even if it's now just one element. Kind of makes sense.